SparkPost/2web2ui
# AC-1298: Changed manually billed condition to look at billing subscri…

### What Changed
 - Changed condition for manually billed to look at subscription type

### How To Test
Via Holmes for most steps
 - Create a free account, and set to manually billed (Holmes)
 - Add R to subscription (Holmes)
 - Change the message bundle to free (Holmes)
- Verify that RV pages do not request for Credit card
